index.js

```diff
@@ -1,18 +1,18 @@
 "use strict";
-const { parse, compileScript, compileTemplate } = require('@vue/compiler-sfc');
 
 (function () {
-  window.translateSFC = async function (source) {
+  window.translateSFC = function (source, convertTS = true) {
```

Comment:
Introduce the "tsCombilerOptions" parameter (to make it possible to pass custom compiler options from a caller) and propagate it to a related ts.transpileModule method call (keep your default options, if the parameter is not specified). I also do not like the "convertTS" parameter name, but I do not have an idea how to name it better.